sdk/tests/conformance2/reading/format-r11f-g11f-b10f.html

```diff
@@ -95,25 +114,34 @@
   clearAndVerifyColor(width, height, true, false, false, 1000);
   clearAndVerifyColor(width, height, true, false, false, 2500);
   clearAndVerifyColor(width, height, true, false, false, 5000);
-  clearAndVerifyColor(width, height, true, false, false, 8191);
-  clearAndVerifyColor(width, height, true, false, false, 8192);
```

Comment:
Sorry, I am not aware of the slight difference between Inf and NaN on Mac when I run this pr. Glad to see that you have added 8192 into the test values. @zhenyao